recipe/meta.yaml

```diff
@@ -10,11 +10,10 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches: 
-    - ngl_fix.patch
 
 build:
-  number: 3
+  noarch: python
+  number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
this may need to be pip for noarch https://conda-forge.org/docs/meta.html#use-pip